cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,57 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import ArticleEditPageObject from '../support/pages/articleEdit.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
 
-describe('Article', () => {
-  before(() => {
+const ArticleEditPage = new ArticleEditPageObject();
```

Comment:
Uppercase is for class names, and lower cases is for objects created from this class
```const articleEditPage = new ArticleEditPageObject();```